vgen/src/main/java/io/gapi/vgen/ruby/RDocCommentFixer.java

```diff
@@ -28,15 +31,40 @@
   private static final Pattern CLOUD_LINK_PATTERN = Pattern.compile(
       "\\[([^\\]]+)\\]\\(([^\\)]+)\\)");
   private static final Pattern PROTO_LINK_PATTERN = Pattern.compile(
-      "\\[([^\\]]+)\\]\\[[^\\]]+\\]");
+      "\\[([^\\]]+)\\]\\[[^\\]]*\\]");
```

Comment:
Do you mind updating the Python as well? I know it's not strictly related. I still think we should eventually define these in a common location since they're cross-language.